test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -379,6 +381,25 @@ void test_ucp_wireup::waitall(std::vector<void*> reqs)
     }
 }
 
+bool test_ucp_wireup::ep_iface_has_caps(ucp_worker_h worker,
```

Comment:
maybe pass ucp_test::entity instead of worker

Reply:
ok, done